code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -57,9 +57,9 @@
 	if(ishuman(C))
 		var/mob/living/carbon/human/H = C
 		if(!pref_load) //Hah! They got forcefully purrbation'd. Force default felinid parts on them if they have no mutant parts in those areas!
-			if(H.dna.features["tail_human"] == "None")
+			if(H.dna.features["tail_human"] != "Cat")
 				H.dna.features["tail_human"] = "Cat"
```

Comment:
Since it's being forced as `"Cat"` every time, you can just remove these checks entirely.
```suggestion
			H.dna.features["tail_human"] = "Cat"
```